openpbs.spec

```diff
@@ -1,626 +0,0 @@
-
```

Comment:
Its used by Travis (to install build nd runtime dependency), its not safe to delete.

Reply:
I was thinking pbspro.spec.in is enough because psbpro.spec is generated from it automatically. It seems pbspro.spec is also used by Travis. So I keep both of them in my next commit.